CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 ## v2.0.3 [unreleased]
 ----------------------
 
+### Features
+
+1. [20128](https://github.com/influxdata/influxdb/pull/20128): Allow password to be specified as a CLI option in `influx v1 auth create`.
+1. [20128](https://github.com/influxdata/influxdb/pull/20128): Allow password to be specified as a CLI option in `influx v1 auth set-password`.
```

Comment:
@gshif this look good to you, won't cause pain when you prep the release later? I kept the text from the original PR and just updated the numbers.

Reply:
Ok